bigbluebutton-tests/playwright/user/lockViewers.js

```diff
@@ -114,16 +115,17 @@ class LockViewers extends MultiUsers {
     await this.modPage.waitAndClick(`${e.unlockUserButton}>>nth=1`);
 
     await this.modPage.waitAndClick(`${e.userListItem}>>nth=0`);
-    await this.modPage.waitAndClick(`${e.unlockUserButton}>>nth=1`);
+    await this.modPage.waitAndClick(`${e.unlockUserButton}>>nth=0`);
     
     await this.userPage2.waitAndClick(`${e.userListItem}>>nth=1`);
     await this.userPage2.waitAndClick(`${e.startPrivateChat}>>nth=1`);
     await this.userPage2.type(e.chatBox, 'Test');
     await this.userPage2.waitAndClick(e.sendButton);
     
+    await this.userPage.waitAndClick(e.closePrivateChat);
     await this.userPage.waitAndClick(`${e.chatButton}>>nth=1`);
-    await this.userPage.hasElementDisabled(e.chatBox);
-    await this.userPage.hasElementDisabled(e.sendButton);
+    await this.userPage.hasElementEnabled(e.chatBox);
+    await this.userPage.hasElementEnabled(e.sendButton);
```

Comment:
I think we lost the flow of this kind of test at some moment. we basically need to ensure some main steps in all lock viewers tests (writing this down with the intention of keeping it saved somewhere):

- join mod
- join user1
- lock viewers
- join user2
- ensure user2 joined locked
- unlock user1
- user2 keeps locked but user1

I've tried the following code snippet locally and it works pretty fine

```js
  async lockSendPrivateChatMessages() {
    const lastUserItemLocator = this.userPage.getLocatorByIndex(e.userListItem, -1);
    await this.userPage.clickOnLocator(lastUserItemLocator);
    const startPrivateChatButton = this.userPage.getLocatorByIndex(e.startPrivateChat, -1);
    await this.userPage.clickOnLocator(startPrivateChatButton);
    await openLockViewers(this.modPage);
    await this.modPage.waitAndClickElement(e.lockPrivateChat);
    await this.modPage.waitAndClick(e.applyLockSettings);
    await this.initUserPage2();
    await this.modPage.getLocator(e.userListItem).filter({ hasNotText: this.userPage2.username }).click();
    await this.modPage.waitAndClick(e.unlockUserButton);

    await this.userPage.getLocator(e.userListItem).filter({ hasText: this.userPage2.username }).click();
    await this.userPage.waitAndClick(`${e.startPrivateChat} >> visible=true`);
    await this.userPage.hasElementEnabled(e.chatBox);
    await this.userPage.hasElementEnabled(e.sendButton);
    await this.userPage.type(e.chatBox, 'Test');
    await this.userPage.waitAndClick(e.sendButton);

    await this.userPage2.getLocator(e.chatButton).filter({ hasText: this.userPage.username }).click();
    await this.userPage2.waitForSelector(e.closePrivateChat);
    await this.userPage2.hasElementDisabled(e.chatBox);
    await this.userPage2.hasElementDisabled(e.sendButton);
  }
```